src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -82,6 +82,14 @@ std::string HttpInfo::toJson()
     {
       jh.addString("payload", this->payload);
     }
+    else if (this->includePayload)
+    {
+      jh.addBool("includePayload", false);
+    }
+    else
+    {
+      jh.addBool("includePayload", true);
+    }
```

Comment:
I'd say that it should be the other way around:

```suggestion
    else if (this->includePayload)
    {
      jh.addBool("includePayload", true);
    }
    else
    {
      jh.addBool("includePayload", false);
    }
```

I mean, if `this->includePayload` is `true` then you have to add `"includePayload": true` in the JSON payload. Or maybe I'm missing something? :)